pkg/printers/internalversion/describe.go

```diff
@@ -2650,6 +2653,18 @@ func getPodStatusForController(c coreclient.PodInterface, selector labels.Select
 	return
 }
 
+// isControllerRef returns true if a pod is managed by a controller or if owner
+// reference is not set; otherwise, return false.
```

Comment:
I'm returning `true` here mainly for backward compatibility; didn't realize orphaned pod case.  If a pod is orphaned, will we be able to select it?  All pods passed in there are selected via controller's selector.